tests/storage/disk_preallocation/test_disk_preallocation.py

```
@@ -0,0 +1,62 @@
+import pytest
+
+from tests.storage.disk_preallocation.utils import (
+    assert_preallocation_annotation,
+    assert_preallocation_requested_annotation,
+)
+
+pytestmark = pytest.mark.post_upgrade
+
+
+@pytest.mark.polarion("CNV-5512")
+@pytest.mark.gating()
```

Ruff (0.15.17)

[warning] Use `@pytest.mark.gating` over `@pytest.mark.gating()`

Remove parentheses

(PT023)